tests/h/presenters/group_json_test.py

```diff
@@ -90,6 +101,17 @@ def test_it_expands_organizations(self, factories, GroupContext, OrganizationJSO
 
         assert model['organization'] == OrganizationJSONPresenter(group_context.organization).asdict.return_value
 
+    def test_expanded_organizations_None_if_missing(self, factories, GroupContext, OrganizationJSONPresenter):  # noqa: N803
```

Comment:
I wonder if a longer test name is possible like `test_expanded_organizations_is_None_if_group_has_no_organization`. Just because it's not clear _what_ is missing in this test name. Alternatively a docstring.